ntoskrnl/mm/section.c

```diff
@@ -3847,30 +3847,28 @@ NtQuerySection(
     {
         case SectionBasicInformation:
         {
-            SECTION_BASIC_INFORMATION Sbi;
+            SECTION_BASIC_INFORMATION Sbi = { 0 };
 
             Sbi.Size = Section->SizeOfSection;
             Sbi.BaseAddress = (PVOID)Section->Address.StartingVpn;
 
-            Sbi.Attributes = 0;
             if (Section->u.Flags.File)
                 Sbi.Attributes |= SEC_FILE;
             if (Section->u.Flags.Image)
                 Sbi.Attributes |= SEC_IMAGE;
-
-            /* Those are not set *************
```

Comment:
NOTE: These were disabled in this commit https://github.com/reactos/reactos/commit/543112008aafc40b34fd68b43baa04645aeaaaf1 .